test/JDBC/expected/BABEL-4161-vu-verify.out

```diff
@@ -0,0 +1,24 @@
+create table #t4161(c1 int, c2 int)
```

Comment:
I really think we should test this more. Different types of columns, temp tables in Functions with primary keys, constraints, etc.